Source/Scene/GroundPrimitive.js

```diff
@@ -425,7 +425,7 @@ define([
 
     function getComputeMinimumHeightFunction(primitive) {
         return function(granularity, ellipsoid) {
-            return primitive._minHeight;
+            return primitive._minHeight - 15;
```

Comment:
Ah okay.  Well that kind of goes beyond my realm of knowledge, so someone else will have to take over here.